src/pages/physical-person.tsx

```diff
@@ -80,7 +84,9 @@ const PhysicalPerson = () => {
     try {
       await physicalPersonService.save(physicalPerson)
       setLoading(false)
-      notifySuccess(PHYSICAL_PERSON_NEW_SUCCESS)
+      const notifyMessage =
+        physicalPerson.id != null ? PHYSICAL_PERSON_EDIT_SUCCESS : PHYSICAL_PERSON_NEW_SUCCESS
```

Comment:
Confere se o id é opcional em physicalPerson, se for, ele pode ser undefined e vc nao precisa comparar com null. Typescript eh bom pois podemos usar undefined e atributos opcionais nas interfaces, isso facilita a verificao. Quano um atributo nao deve existir, ele eh undefined ou inves de null. Usamos null para coisas mais especificas.

Reply:
O id é opcional, removi a verificação com null